cedar-lean/DiffTest/Parser.lean

```diff
@@ -463,7 +475,7 @@ partial def jsonToEntityOrRecordType (json : Lean.Json) : ParseResult CedarType
     let lub ← Array.mapM jsonToName lubArr
     if lub.size == 1
     then .ok (.entity lub[0]!)
-    else .error s!"jsonToEntityOrRecordType: expected lub to have exactly one element¬{json.pretty}"
```

Comment:
😆

Reply:
I love the unicode support for Lean in VSCode! But it's a little over-eager sometimes